performance/multi-node/rdm_mbw_mr.c

```diff
@@ -571,6 +698,24 @@ int main(int argc, char *argv[])
 	hints->ep_attr->type	= FI_EP_RDM;
 	hints->caps		= FI_MSG | FI_DIRECTED_RECV;
 	hints->mode		= FI_CONTEXT | FI_LOCAL_MR;
+	hints->domain_attr->control_progress = FI_PROGRESS_AUTO;
+	hints->domain_attr->data_progress = FI_PROGRESS_AUTO;
+	if (myid == 0) {
+		if (test_type == CQ_TEST)
+			fprintf(stderr, "Test will use CQ Completions\n");
+
+		if (test_type == CNTR_TEST)
+			fprintf(stderr, "Test will use CNTR Completions\n");
+
+		if (wait_type == FI_WAIT_NONE)
+			fprintf(stderr, "WaitObject type is FI_WAIT_NONE\n");
+
+		if (wait_type == FI_WAIT_UNSPEC)
+			fprintf(stderr, "WaitObject type is FI_WAIT_UNSPEC\n");
+
+		if (wait_type == FI_WAIT_SET)
```

Comment:
In my haste to clean this up, that thought did not occur to me. I had originally located all these fprintf's in different locations, but moved them all here. I'll change this to a switch when I address any other needed changes.